api/v1alpha4/types.go

```diff
@@ -137,6 +137,11 @@ type PortOpts struct {
 	// The virtual network interface card (vNIC) type that is bound to the neutron port.
 	VNICType string `json:"vnicType,omitempty"`
 
+	// A dictionary that enables the application running on the specified
+	// host to pass and receive virtual network interface (VIF) port-specific
+	// information to the plug-in.
+	Profile map[string]interface{} `json:"profile,omitempty"`
```

Comment:
```suggestion
	Profile map[string]string `json:"profile,omitempty"`
```

I think you have to pass a map of strings here and convert it later on to `map[string]interface{}` via

```go
var portProfile map[string]interface{}
for k, v := range portOpts.Profile {
	portProfile[k] = v
}
```

Reply:
indeed. Thank you!